Templates/Applications/Salesforce.json

```diff
@@ -414,6 +414,26 @@
       "width": "medium",
       "pad": ":::small"
     },
+    "initiationTypeInfo": {
+      "staticType": "fieldset",
+      "content": " - ",
+      "before": [
+        "initiationTypeLabel",
+        {
+          "name": "initiationType",
+          "noFieldset": true,
+          "inline": true
+        }
+      ],
+      "after": {
+        "name": "initiationBy",
+        "inline": true
+      },
+      "markdown": "inline"
```

Comment:
The `markdown` property of a static element doesn't actually do anything anymore.  This can only be handled through the property flags.

Reply:
Oh i just copied this from the SAML one to have some consistency, i can fix it on both